adal/src/androidTest/java/com/microsoft/aad/adal/DiscoveryTests.java

```diff
@@ -311,6 +311,51 @@ public Void call() throws Exception {
         Mockito.verify(mockedConnection, Mockito.times(1)).getInputStream();
     }
 
+    /**
+     * Verified scenario:
+     * When an authority is valid and metadata is returned:
+     * a. Subsequent requests for any aliases provided in the metadata do not result in further validation network requests
+     * being made for the process lifetime
+     */
+    @Test
+    public void testAuthorityInAliasedList() throws IOException {
+        AuthorityValidationMetadataCache.clearAuthorityValidationCache();
```

Comment:
Nit: Seems more appropriate to place in a `setUp()` function

Reply:
done.